#150798549 Nag user until they have resolved an issue

Annettesunday opened the pull request:
#### What does this PR do?
Nag user by email until they have resolved an issue
#### Description of Task to be completed?
Configure nag user intervals
Notify the user over and over until they have resolved the issue

#### How should this be manually tested?
After clowning the repo,cd into it and run: 

- python manage.py makemigrations 
- python manage.py migrate
- python manage.py test

#### What are the relevant pivotal tracker stories?

#### Screenshot

![image](https://user-images.githubusercontent.com/26150162/30384814-0325f4ee-98ae-11e7-8158-e084090dab43.png)

#### Questions:
##### Why isn't there an explicit icon on the dashboard showing the nag mode
Nagging a user is only active when the check is down. There is already an icon depicting when a check is down. That can double as a nag icon too.

Annettesunday committed:
feat(Sendalerts): Implement email-nagging functionality
- add two columns to Check model
- modify nag time and nag interval
- make and apply migrations

Annettesunday committed:
feat(Sendalerts): Implement email-nagging functionality
- add two columns to Check model
- modify nag time and nag interval
- make and apply migrations

muhallan committed:
style(sendalerts): add comments to changes made
- add comments to reflect changes made to implement nagging alerts

muhallan committed:
style(sendalerts): add comments to changes made
- add comments to reflect changes made to implement nagging alerts

muhallan committed:
feat(nag interval): add the UI for modifying the nag interval
- create a new slider that changes nag intervals
- display the selected nag interval on the checks table

muhallan committed:
feat(nag interval): add the UI for modifying the nag interval
- create a new slider that changes nag intervals
- display the selected nag interval on the checks table

muhallan committed:
fix(migrations): delete duplicate migrations
- delete migrations that were clashing with each and yet did the same thing

muhallan committed:
fix(migrations): delete duplicate migrations
- delete migrations that were clashing with each and yet did the same thing

muhallan committed:
test(update timeout): test for nag time
- refactor tests for timeout to include the expected nag time

muhallan committed:
test(update timeout): test for nag time
- refactor tests for timeout to include the expected nag time

judithachieng commented:
`Notify the user over and over until they have resolved the issue` I believe this should have been phrased in a better way @Annettesunday

judithachieng commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": false
+}
```
.vscode should not be pushed to github.

judithachieng replied:
please remove them @Annettesunday @muhallan

muhallan committed:
refactor(.gitignore): remove .vscode from remote
 - delete the now ignored .vscode folder

muhallan committed:
Merge branch 'ft-continuous-notification-150798549' of https://github.com/andela/healthchecks-invincibles into ft-continuous-notification-150798549
This is to fetch the remote changes.

muhallan committed:
refactor(.gitignore): remove .vscode
- delete the now ignored folder .vscode

muhallan committed:
feat(nag user): add nag icon to the dashboard
 - add an icon on a check indicating system sending of nag alerts
 - create a custom email format to be sent for nagging alerts
 - add nag icon in the docs
